charts/spire/charts/spire-server/README.md

```diff
@@ -62,6 +62,8 @@ A Helm chart to install the SPIRE server.
 | jwtIssuer | string | `"oidc-discovery.example.org"` |  |
 | logLevel | string | `"info"` |  |
 | nameOverride | string | `""` |  |
+| nodeAttestor.k8s.enabled | bool | `true` |  |
+| nodeAttestor.k8s.rbac.create | bool | `true` |  |
```

Comment:
Can you add psat in here? cc @marcofranssen

Reply:
Do we plan on supporting non psat with the template?

Reply:
Probably not sat attestor but i think its good to be unambiguous that this relates to the psat attestor.

Reply:
kk

Reply:
done